### PR TITLE
Populate RSV form if user is already logged in

### PR DESCRIPTION
**Pull request by mitogh**
**Ticket** https://central.tri.be/issues/78607

### CODE REVIEW

**Comment by mitogh** [comment]
**Note** still needs changelog as we need to setup the target branch with the appropriate version and readme changes.

**Comment by barryhughes** [comment]
I (just) added a section in the changelog for M18.05 code (version number isn't defined yet, just reads as `M18.05`)

**Comment by mitogh** [comment]
Perfect will sync with `base` branch, thanks.

File: src/views/tickets/rsvp.php [hunk 1]
```diff
@@ -127,22 +128,34 @@ class="tribe-ticket-quantity"
 					 * @var array of Tribe__Tickets__Ticket_Object
 					 */
 					do_action( 'event_tickets_rsvp_before_confirmation_fields', $tickets );
+
+					$name = '';
+					$email = '';
+					if ( is_user_logged_in() ) {
+						$current_user = wp_get_current_user();
+						$name_parts = array( $current_user->first_name, $current_user->last_name );
+						$name = implode( ' ', array_filter( $name_parts ) );
```
[the remark this answers is not in the document]
Comment (mitogh) [reply src/views/tickets/rsvp.php]: I think it makes more sense, however we still need to add the filters into the template tho, but I agree having a flexibility to rollback with filters give us more space to move around.

Thanks for the suggestion and pointing that out.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $default_email is never used | [RSVP.php:232](https://github.com/moderntribe/event-tickets/blob/3b23836a817dc43d9fb81e5f89e90f3a48e147f8/src/Tribe/RSVP.php#L232)

> via jenkins/codesniffer-phpcs

**Comment by zbtirrell** [comment]
this is dangerously close to Tribe Commerce changes

**Comment by mitogh** [comment]
Removing the `hold` as we have the changes merged.

**Comment by mitogh** [comment]
Waiting for other PRs to be merged first to fix the test on this one.

**Pull request merged**